.github/workflows/build.yml

```
@@ -50,3 +50,8 @@ jobs:
           firefox-version: "84.0"
       - run: |
           firefox --version
+      - uses: ./
+        with:
+          firefox-version: "devedition-84.b1"
```

The test [fails](https://github.com/browser-actions/setup-firefox/actions/runs/7716884084/job/21678745839#step:9:8).  I think it should be `devedition-84.0b1`

```suggestion
          firefox-version: "devedition-84.0b1"
```